client/ayon_ftrack/plugins/publish/integrate_hierarchy_ftrack.py

```diff
@@ -287,6 +287,13 @@ def import_to_ftrack(
             instances = []
             for instance in context:
                 instance_folder_path = instance.data.get("folderPath")
+
+                # Ensure project name is the first entry in the folder path.
+                if not instance_folder_path.startswith("/" + project_name):
```

Comment:
But `entity_path` should not contain project name. If it does, then we should make sure it does not...

Reply:
Rather change [this line](https://github.com/ynput/ayon-ftrack/pull/88/files#diff-62a6c74c35c702c33b39c835c1846ecaf770a1735df0b12fd6e5a0a4b88f8509R283) to.
```python
            if entity_type.lower() == "project":
                entity_path = ""
            else:
                entity_path = "{}/{}".format(parent_path, entity_name)
```

Reply:
ok, will investigate where the problem is.

Reply:
The suggested change should fix any issues you hit.

Reply:
> Rather change [this line](https://github.com/ynput/ayon-ftrack/pull/88/files#diff-62a6c74c35c702c33b39c835c1846ecaf770a1735df0b12fd6e5a0a4b88f8509R283) to.
> 
> ```python
>             if entity_type.lower() == "project":
>                 entity_path = ""
>             else:
>                 entity_path = "{}/{}".format(parent_path, entity_name)
> ```

Which line? The link goes a diff.

Reply:
Currently on line 283 there is
```
entity_path = "{}/{}".format(parent_path, entity_name)
```